src/Http/Controllers/Auth/SocialiteController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers\Auth;
+
+use Blumilk\Meetup\Core\Http\Controllers\Controller;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Http\RedirectResponse;
+use Laravel\Socialite\Facades\Socialite;
+
+class SocialiteController extends Controller
+{
+    public function redirectToGoogle(): RedirectResponse
+    {
+        return Socialite::driver("google")->redirect();
+    }
+    public function handleGoogleCallback(): RedirectResponse
+    {
+        $user = Socialite::driver("google")->user();
+
+        $this->registerOrLogin($user);
+
+        return redirect()->route("home");
+    }
```

Comment:
```suggestion
    }
    
```
Please can you add a space after the method.